contextvh/src/main/java/contextvh/translators/J2PopupData.java

```diff
@@ -69,8 +69,10 @@ public J2PopupData() { }
 
         MultiPolygon mpolygon = popup.getMultiPolygon();
         Parameter parPolygon = new Identifier("NO_MULTIPOLYGON");
+        Parameter parAreaSize = new Identifier("NO_SIZE");
```

Comment:
Wouldn't it be better to set the default value on -1 or something? Using identifiers and numerals interchangeably in one variable doesn't seem like a good idea.